### PR TITLE
V33 fix dss script

### PR DESCRIPTION
**Pull request by aledj2**
This change is [Reviewable](https://reviewable.io/reviews/moka-guys/automate_demultiplex/294)

### CODE REVIEW

File: automate_demultiplex_config.py [hunk 1]
```diff
@@ -17,9 +17,9 @@
 document_root = "/".join(os.path.dirname(os.path.realpath(__file__)).split("/")[:-2])
 
 # # path to run folders
-runfolders = "/media/data3/share"
+#runfolders = "/media/data3/share"
 # when testing use a different directory
-#runfolders = "/media/data3/share/testing"
+runfolders = "/media/data3/share/testing"
```
Comment (natashapinto) [inline automate_demultiplex_config.py:22]: Is it meant to point to a test folder?

**Review by natashapinto — CHANGES_REQUESTED** [review]
Just a comment on where the runfolder path is pointing to

**Review by aledj2 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/moka-guys/automate_demultiplex/294)* status: 0 of 3 files reviewed, 1 unresolved discussion (waiting on @natashapinto)

___
*[automate_demultiplex_config.py, line 22 at r1](https://reviewable.io/reviews/moka-guys/automate_demultiplex/294#-MLE3ba77jkDWpCybwSW-r1-22:-MLI1Z-J6q00xxFjo8QD:b-896fix) ([raw file](https://github.com/moka-guys/automate_demultiplex/blob/10cbec787eab49ffce94f4f9eac6f9358a053764/automate_demultiplex_config.py#L22)):*
<details><summary><i>Previously, natashapinto wrote…</i></summary><blockquote>

Is it meant to point to a test folder?
</blockquote></details>

Done.

**Review by natashapinto — APPROVED** [review]
> Reviewed 2 of 3 files at r1, 1 of 1 files at r2.
> *[Reviewable](https://reviewable.io/reviews/moka-guys/automate_demultiplex/294)* status: :shipit: complete! all files reviewed, all discussions resolved

**Pull request merged**